### ISSUE TITLE
- instead of _

### ISSUE THREAD

**Issue by lorenblue** [p6laris/MorseSharp#30]
I am not familiar with morse code, but my understanding is that the dash is usually represented as "-" as opposed to "_". Could this be an option? If not, I guess I can just replace the underscores with dashes after encoding for now.

**Comment by p6laris** [comment]
Yes You’re absolutely right, traditionally dashes are represented as ‘-’. I’ll work on fixing that as soon as possible and standardize it. 

Thank you for pointing that out.
